larray/inout/xw_excel.py

```diff
@@ -806,19 +809,15 @@ def open_excel(filepath=None, overwrite_file=False, visible=None, silent=None, a
 
 Examples
 --------
->>> arr = ndtest((3, 3))
->>> arr
-a\b  b0  b1  b2
- a0   0   1   2
- a1   3   4   5
- a2   6   7   8
+>>> arr, arr2 = ndtest((3, 3)), ndtest((2, 2))
 
 create a new Excel file and save an array
 
 >>> # to create a new Excel file, argument overwrite_file must be set to True
 >>> with open_excel('excel_file.xlsx', overwrite_file=True) as wb:   # doctest: +SKIP
 ...     wb['arr'] = arr.dump()
-...     wb['arr']['A6'].add_plot('A1', title='simple graph')
+...     wb['arr']['A6'].add_table()
+...     wb['arr']['G1'].add_plot('A1', title='simple graph')
```

Comment:
This probably works, but I personally prefer using explicit data range instead of just the top cell. Both should be shown in the doctest though I think, because it's nice to know that it works.